Tests/tinys3Tests/requests/ListMultipartUploadsRequestTests.swift

```diff
@@ -0,0 +1,66 @@
+import XCTest
+@testable import tinys3
+
+// swiftlint:disable line_length
+final class ListMultipartUploadsRequestTests: XCTestCase, RequestTest {
+    let request = AWSRequest.listMultipartUploadsRequest(
+        bucket: "examplebucket",
+        key: "images/xcode-14.3.vmtemplate.aar",
+        credentials: .testDefault,
+        date: .testDefault
+    )
+
+    func testThatCanonicalUriIsCorrect() throws {
+        XCTAssertEqual("/", request.canonicalUri)
+    }
+
+    func testThatCanonicalQueryStringIsCorrect() throws {
+        XCTAssertEqual("prefix=images%2Fxcode-14.3.vmtemplate.aar&uploads=", request.canonicalQueryString)
+    }
+
+    func testThatCanonicalHeaderStringIsCorrect() throws {
+        XCTAssertEqual("""
+host:examplebucket.s3.amazonaws.com
+x-amz-content-sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+x-amz-date:20130524T000000Z
+""", request.canonicalHeaderString)
```

Comment:
Nitpick: indent the multiline string to be one level deep than `XCTAssertEqual`?

```
swift repl
Welcome to Apple Swift version 5.10 (swiftlang-5.10.0.13 clang-1500.3.9.4).
Type :help for assistance.
  1> let string = """
  2.     hello
  3.     world
  4.     """
string: String = "hello\nworld"
  5> print(string)
hello
world
```

Reply:
Good point. Fixed in 342de5de7cf7477ca2f30bd1699049d3c49c434a